agent/src/capability/capability.cpp

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
You mean to move some of the functionality I currently have in configstore, or to add functionality related to merging capabilities?